CMakeLists.txt

```diff
@@ -324,6 +330,9 @@ if(UNIX)
   elseif (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
       set(LIBTUNTAP_IMPL ${TT_ROOT}/tuntap-unix-darwin.c ${TT_ROOT}/tuntap-unix-bsd.c)
       get_filename_component(EV_SRC "llarp/ev/ev_kqueue.cpp" ABSOLUTE)
+  elseif (IOS)
```

Comment:
See the three lines above.  The implementation for Darwin platforms (which iOS is one of) is the same.

Prior to cmake 3.14, compiling for iOS has `CMAKE_SYSTEM_NAME` set to `Darwin`.  3.14+ have better support for mobile Darwin platforms, and set it to `iOS`, `watchOS`, etc.  See [here](https://github.com/leetal/ios-cmake/blob/3e5697cd87ca7882783ad3a22c86369256f69f6d/ios.toolchain.cmake#L420), lines 420 to line 444 inclusive.

Depending on maintainer preference, and the cmake versions we're using, probably should remove the lines 333-335 you've added, and amend line 330 to read
```cmake
  # this is a standard flag used in the iOS, macOS, etc, toolchains
  elseif (APPLE)
```
or
```cmake
  # prior to cmake 3.14, iOS/tvOS/watchOS set this to Darwin
  # cmake 3.14+ set it to iOS, tvOS, etc as appropriate
  elseif(${CMAKE_SYSTEM_NAME} MATCHES "(Darwin|(i|tv|watch)OS)")
```